### ISSUE TITLE
Improve neural network

### ISSUE THREAD

**Issue by kiosklabs** [kiosklabs/kaggle-digit-recognizer-competition#7]
Our latest submission only reaches 0.98 validation accuracy.
Let's look into creating a learning curve for the training data and check if using SVC would improve the results.
# Neural Network with 160362 learnable parameters
## Layer information

  #  name      size

---

  0  input     1x28x28
  1  conv1     32x24x24
  2  pool1     32x12x12
  3  conv2     32x8x8
  4  pool2     32x4x4
  5  dense     256
  6  dropout2  256
  7  output    10

  epoch    trn loss    val loss    trn/val    valid acc  dur

---

```
  1     2.59348     1.59637    1.62461      0.44128  198.98s
  2     1.34441     0.83408    1.61184      0.71517  200.59s
  3     0.73780     0.40196    1.83552      0.86960  204.79s
  4     0.40493     0.22051    1.83635      0.93349  199.86s
  5     0.26865     0.17966    1.49531      0.94432  198.29s
  6     0.20499     0.13746    1.49127      0.96062  202.80s
  7     0.17579     0.12504    1.40589      0.96264  197.96s
  8     0.15123     0.11379    1.32899      0.96538  202.97s
  9     0.13884     0.11433    1.21438      0.96692  195.14s
 10     0.12960     0.13178    0.98352      0.96324  198.04s
 11     0.11432     0.11997    0.95287      0.96633  199.35s
 12     0.11058     0.10036    1.10184      0.96954  214.94s
 13     0.10577     0.09770    1.08263      0.97097  197.59s
 14     0.09661     0.09389    1.02894      0.97204  205.60s
 15     0.09026     0.08589    1.05082      0.97513  199.08s
 16     0.08384     0.09447    0.88744      0.97275  206.67s
 17     0.08292     0.08354    0.99249      0.97609  194.11s
 18     0.08391     0.08469    0.99084      0.97573  195.81s
 19     0.07493     0.07813    0.95900      0.97597  197.96s
 20     0.07873     0.07663    1.02743      0.97775  199.85s
 21     0.06879     0.08042    0.85533      0.97632  203.46s
 22     0.06836     0.08184    0.83532      0.97739  205.68s
 23     0.06542     0.07457    0.87724      0.97882  205.72s
 24     0.06603     0.07917    0.83399      0.97751  209.25s
 25     0.06369     0.08106    0.78564      0.97704  205.78s
 26     0.05967     0.07484    0.79728      0.97728  205.74s
 27     0.05882     0.07083    0.83048      0.98013  205.87s
 28     0.05776     0.08018    0.72047      0.97656  198.86s
 29     0.05429     0.08168    0.66465      0.97847  198.53s
 30     0.05224     0.07266    0.71891      0.98025  196.38s
```
